pandas/pyproject.toml

```diff
@@ -0,0 +1,34 @@
+# Note: This is the pyproject.toml for the pandas-tests package
+# The pyproject.toml for the pandas package is in the parent directory
+# of this one
+[build-system]
+requires = [
+    "setuptools>=61.0.0",
+]
+[project]
+name="pandas-tests"
+# Note: The version is hardcoded since the pandas version has to
+# match the tests version. Remember to bump both!
+dynamic = ["version"]
+#dependencies=[
+#    "pandas==2.1.0"
+#]
```

Comment:
Should this still be commented out?

Reply:
Thanks for the reminder.

I planned on doing this in a followup (since technically this only needs to be set at release time), but then realized I can do this programmtically.

I'll make the changes to the wheel builder workflow soon.